README.md

```diff
@@ -97,10 +97,16 @@ npm run lint:fix
 
 ### testing
 
+Copy phpunit.xml.example to phpunit.xml and fill in the test database credentials or leave it as is for in-memory testing with sqlite.
```

Comment:
@TobiasGrether do you think it would make sense to update the dependencies and libs to support testing with [Laravel Pest](https://pestphp.com/)? (I think this is the modern solution instead of phpunit)

Reply:
@simonostendorf PHPUnit is generally decent, I've only used it for static code analysis so far. But yeah Laravel Pest would probably be a better choice. API looks nice

Reply:
I tried migrating to pest, but the tests won't run due to problems with the faker in factories. I have no idea how to fix this.